README.md

```diff
@@ -113,6 +113,8 @@ In this example, `main.cpp` defines the update binary for the bootloader as `mbe
 
 The next step is to build an application you can combine with your bootloader to create a loadable image.
 
+<span class="tips">**Note:** Building an application which uses a bootloader currently does not work with the online compiler since bin and hex files cannot be uploaded.</span>
```

Comment:
Please change to "... Building an application that uses a bootloader currently doesn't work with the Arm Mbed Online Compiler because .bin and .hex files can't be uploaded."

Reply:
This is simply not true. Converting an imported application to using a bootloader is what's broken.

Reply:
> Converting an application which to use a bootloader currently does not work with the online compiler since bin and hex files cannot be uploaded

Reply:
Updating an an application to use a bootloader currently does not work with the Arm Mbed Online Compiler because .bin and .hex files can't be uploaded.

Reply:
Sure, with one an.